src/app/scripts/cac/places/cac-places.js

```diff
@@ -79,77 +79,57 @@ CAC.Places.Places = (function(_, $, moment, Routing, UserPreferences) {
         return otpOptions;
     }
 
-    /** Get travel times from the orign to each destination.
+    /** Get distance (as the crow flies) from the orign to each destination.
      *  Should be called before displaying place list.
      *
      * @param destinations {Array} Destination and event objects that will be changed in-place
-     * @param exploreLatLng {Array} Coordinates of origin point to query with; may be unset
-     * @returns {Array} Collection of promises, each of which resolve to a destination
+     * @param exploreLatLng {Array} Coordinates of origin point to start from; may be unset
+     * @returns {Array} Destinations with distance and orign label set
      */
-    function getTimesToPlaces(destinations, exploreLatLng) {
-        // if origin not set, simply clear the travel times from the destinations
-        if (!exploreLatLng) {
-            _.each(destinations, function(destination) {
-                destination.duration = undefined;
-                destination.formattedDuration = undefined;
-                destination.originLabel = undefined;
-            });
-            return;
-        }
-
-        // make ajax requests to get the travel times to each destination
-        var otpOptions = getOtpOptions();
-        // only using the first itinerary; let OTP know to not bother finding other options
-        $.extend(otpOptions, {numItineraries: 1});
-
-        var date = UserPreferences.getPreference('dateTime');
-        date = date ? moment.unix(date) : moment(); // default to now
-
+    function getDistancesToPlaces(destinations, exploreLatLng) {
         // prefer to label with just the street address, and fall back to full address
         // for featured places, use the label in the 'name' attribute instead of the address
         var origin = UserPreferences.getPreference('origin');
-        var originLabel = origin.name ? origin.name :
-            (origin.attributes && origin.attributes.StAddr ? origin.attributes.StAddr :
-            UserPreferences.getPreference('originText'));
+        var originLabel;
+
+        if (origin) {
+            originLabel = origin.name ? origin.name :
+                (origin.attributes && origin.attributes.StAddr ? origin.attributes.StAddr :
+                UserPreferences.getPreference('originText'));
+        }
+
+        if (exploreLatLng) {
+            var from = turf.point(exploreLatLng);
```

Comment:
Looks like [turf-point wants `(lon, lat)` pairs](https://www.npmjs.com/package/turf-point), not the other way around.  This explains why it's saying my house is only 1.1 miles from Fairmount Waterworks--at -75° latitude, a .08° longitude difference is not so far.